src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.box {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
```

Comment:
this styles for header, you don't need box at all, also add to header left and right paddings

Reply:
fix this comment